### ISSUE TITLE
Improve installation instructions

### ISSUE THREAD

**Issue by mstimberg** [eferrea/BCI-Master#5]
[Opened as part of JOSS review openjournals/joss-reviews#3956]

Consider improving installation instructions
Going through with the installation was non-trivial, in particular since it requires several external dependencies that need to be installed first, and no help is given besides links to their websites.
 I know that Visual Studio projects are a common way of setting up things in Windows, but maybe either clearly state that you focus on Windows, or use a more general build system (e.g. `cmake`). I was able to build things under Linux, but I basically ignored everything in the Visual Studio project (which also includes plenty of stuff that seems irrelevant, like log files, etc.), and directly built the files in the `Source Files` folder with:
 ```
 g++ -I/usr/local/include -L/usr/local/lib *.cpp $(pkg-config --libs sfml-all) -lvrpn -lpthread -lquat
 ```
 
 You should also note that the Matlab code requires the "Statistics and Machine Learning Toolbox". Also (but I haven't used Matlab in a long time, so this might be trivial): I had to use `addpath` with `BCI_Classes` etc. to get the `BCI_Loop` function to run.

**Comment by eferrea** [comment]
Hi @mstimberg, I provide now only the source code and a cmake list to build the project under different OS architectures. I tested the project under Windows and Ubuntu lts 20.04. The readme was also modified accordingly on how to use cmake in the different OSs. 
I also added that the Matlab code requires the Statistics and Machine Learning toolbox. Given the few dependencies from this toolbox, I plan to remove them soon. 
Finally, in the readme, I also specify to add the subfolders to the Matlab path.
I hope this can help speeding-up installation procedures.
